packages/runtime/container-runtime/src/test/summary/runningSummarizer.spec.ts

```diff
@@ -669,61 +669,6 @@ describe("Runtime", () => {
 					assert.strictEqual(stopCall, 1);
 				});
 
-				it("Should not retry on failure when stopping instead of restarting", async () => {
```

Comment:
Move this tests to container runtime?

Reply:
Sure, not this exact test. But testing of similar functionality - container runtime should close instead of fetching snapshot. Unless it already exists.